src/pybind/ceph_argparse.py

```diff
@@ -24,7 +24,7 @@
 
 
 FLAG_MGR = 8   # command is intended for mgr
-
+FLAG_POLL = 16
```

Comment:
mind adding a brief comment telling us where the magic number comes from (i.e., mon/MonCommands.h's command flag)? :)

Reply:
Right :) Done.